### PR TITLE
test: full code coverage (#77)

### PR DESCRIPTION
**Pull request by alemagio**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

NOTE:

Please consider I needed to esport the `vary` function in order to test the `else` path which is never taken otherwise.
It is just a way but I'm open to suggestions off course.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -173,3 +173,5 @@ module.exports = fp(fastifyCors, {
   fastify: '3.x',
   name: 'fastify-cors'
 })
+
+module.exports.vary = vary
```
Comment (mcollina) [inline index.js:177]: Can you move `vary` to its own file so you can test it separately without exposing it to the public?

Comment (alemagio) [reply index.js:177]: Is there a reason why we can't use the `vary` function of the `vary` module?

Comment (mcollina) [reply index.js:177]: that works with node core `ServerResponse` object.

Comment (alemagio) [reply index.js:177]: ok, thanks

Comment (alemagio) [reply index.js]: Do you think I should add more tests for the `vary` function itself?

**Pull request merged**